client-common/src/main/java/org/apache/cassandra/sidecar/common/response/data/ActiveCompactionEntry.java

```diff
@@ -0,0 +1,296 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.common.response.data;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.cassandra.sidecar.common.DataObjectBuilder;
+
+/**
+ * Represents an active compaction entry
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ActiveCompactionEntry
+{
+    public static final String ID = "id";
+    public static final String KEYSPACE = "keyspace";
+    public static final String TABLE = "table";
+    public static final String TASK_TYPE = "taskType";
+    public static final String COMPLETED_BYTES = "completedBytes";
+    public static final String TOTAL_BYTES = "totalBytes";
+    public static final String PERCENT_COMPLETED = "percentCompleted";
+    public static final String SS_TABLES = "ssTables";
```

Comment:
It is less common to break up the term `sstable`

```suggestion
    public static final String SSTABLES = "sstables";
```

Reply:
done